Packet++/CMakeLists.txt

```diff
@@ -51,16 +53,17 @@ add_library(
   src/TelnetLayer.cpp
   src/TextBasedProtocol.cpp
   src/TLVData.cpp
+  src/TpktLayer.cpp
   src/UdpLayer.cpp
   src/VlanLayer.cpp
   src/VxlanLayer.cpp
   src/WakeOnLanLayer.cpp
-  # Force hash-library pcapng to be link fully static
```

Comment:
This comment has been involuntary sorted and should be next to $<TARGET_OBJECTS:hash-library>